cmd/globals.go

```diff
@@ -347,8 +347,10 @@ var (
 	globalRootDiskThreshold uint64
 
 	// Used for collecting stats for netperf
-	globalNetPerfMinDuration = time.Second * 10
-	globalNetPerfRX          netPerfRX
+	globalNetPerfMinDuration     = time.Second * 10
+	globalNetPerfRX              netPerfRX
+	globalObjectPerfBucket       = "minio-perf-test-tmp-bucket"
```

Comment:
perhaps we can use `minio` bucket name here? since it is a reserved name and would not conflict with the user created bucket?

Reply:
since `minio` is a reserved bucket I am not sure how the side effects are going to be. Since `warp` had it's own bucket I did it this way

Reply:
The problem is we are deleting this bucket recursively, it is perhaps better to avoid deleting an existing bucket.

We only attempt a delete on a bucket that we created during the test.